doc/source/deploy-guide/configure-environment.rst

```diff
@@ -104,15 +104,15 @@ You can find examples of the required files in `examples/workdir`.
 Configure the VIP that will be used for OpenStack
 -------------------------------------------------
 
-Add `suse_osh_deploy_vip_with_cidr:` with its appropriate value for your
+Add `suse_socok8s_deploy_vip_with_cidr:` with its appropriate value for your
```

Comment:
Based on the references below to this vip variable, I think you meant "socok8s_deploy_vip_with_cidr"? I don't have strong opinion on names, so this would be a nitpick you can ignore: socok8s_deploy even seems to a little redundant, socok8s_vip_with_cidr seems to be pretty clear. Also fyi, I think Scott added multiple vips this afternoon, so we have public/external vip, dcm vip, similar to the names in Cloud 9.

Reply:
In that case, we can probably have 
``socok8s_external_vip_with_cidr``, ``socok8s_dcm_vip_with_cidr``, and so on?

Reply:
We can certainly improve things but it might be better if we do those as post-merge actions. Any improvement (variable name change) in this patch means, that needs to be tested/verified in ucp branch side as well.

Reply:
lgtm then.